tests/conftest.py

```diff
@@ -4,6 +4,7 @@
 import psims
 import pytest
 import yaml
+import math
```

Comment:
**todo:** Use the recommend Python import order:
1. Block with standard library imports.
2. Block with external library imports.
3. Block with local application imports.

An then within each block the imports are ordered alphabetically.

So here, the `import math` should be part of a first block of imports, then an empty line, and then these remaining imports.